docs/source/install/rhel6.rst

```diff
@@ -386,19 +386,19 @@ If you already run Hubot instance, you only have to install the `hubot-stackstor
 
 * That's it! Go to your Chat room and begin ChatOps-ing. Read more in the :doc:`/chatops/index` section.
 
-Upgrade to Enterprise Edition
------------------------------
-Enterprise Edition is deployed as an addition on top of StackStorm Community. You will need an active
-Enterprise subscription, and a license key to access StackStorm enterprise repositories. To add your
-license key, replace ``${ENTERPRISE_LICENSE_KEY}`` in the command below with the key you received when
+Upgrade to Brocade Workflow Composer
+-------------------------------------
+Brocade Workflow Composer is deployed as an addition on top of StackStorm. You will need an active
+Brocade Workflow Composer subscription, and a license key to access Brocade Workflow Composer repositories.
+To add your license key, replace ``${BWC_LICENSE_KEY}`` in the command below with the key you received when
```

Comment:
We may need to update ${ENTERPRISE_LICENSE_KEY} everywhere? Hope you cover it here, but there may be scripts, emails, so let's sniff for changes outside of docs.